ksql-engine/src/main/java/io/confluent/ksql/util/StatementWithSchema.java

```diff
@@ -17,42 +17,41 @@
 package io.confluent.ksql.util;
 
 import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
+import io.confluent.ksql.parser.KsqlParser.PreparedStatement;
 import io.confluent.ksql.parser.SqlFormatter;
 import io.confluent.ksql.parser.tree.AbstractStreamCreateStatement;
 import io.confluent.ksql.parser.tree.Statement;
 
 public final class StatementWithSchema {
-  private final Statement statement;
-  private final String statementText;
 
-  private StatementWithSchema(final Statement statement, final String statementText) {
-    this.statement = statement;
-    this.statementText = statementText;
+  private StatementWithSchema() {
```

Comment:
We have `PreparedStatement` for passing around a statement and text - so switching to using that and making this a utility class.